server/migrations/20170915145623_add_winner_to_raffles.js

```diff
@@ -0,0 +1,12 @@
+
+exports.up = function(knex, Promise) {
+  return knex.schema.table('users', function (table) {
```

Comment:
What is this for?

Reply:
That migration is adding a column to the raffles table so that we can store the winning entry

Reply:
Looks like it's adding it to the `users` table. Maybe I'm wrong.

Reply:
You are so very right 😬 I'll push up a fix!